src/components/Integrations/Shopify/Shopify.js

```diff
@@ -88,30 +86,36 @@ const Shopify = ({ intl, dependencies: { shopifyClient } }) => {
             <>
               <div className="dp-integration__block">
                 {shopifyHeader}
-                <div className="block dp-integration__status">
-                  <div className="status__info">
-                    <div>
-                      <StyledShopifyLogo />
-                      <div className="status__data">
-                        <p>
-                          {_('shopify.header_synchronization_date')} <time>22/06/2019</time>
-                        </p>
-                        <ul>
-                          {shops.map((shop) => (
+                {shops.map((shop) => (
+                  <div className="block dp-integration__status">
+                    <div className="status__info">
+                      <div>
+                        <StyledShopifyLogo />
+                        <div className="status__data">
+                          <ul>
                             <li key={shop.shopName}>
+                              <p>
+                                {_('shopify.header_synchronization_date')}{' '}
+                                <strong>
+                                  <FormattedDate value={shop.synchronization_date} timeZone="UTC" />
+                                </strong>
+                              </p>
                               <p>
                                 {_('shopify.header_store')} <strong> {shop.shopName} </strong>
                               </p>
                             </li>
-                          ))}
-                        </ul>
+                          </ul>
+                        </div>
                       </div>
                     </div>
+                    <a
+                      href={'https://' + shop.shopName + '/admin/apps'}
```

Comment:
What do you think about including this in the language files?

Reply:
Yes it could be.

Reply:
How should I handle things here with tagged template functions? mmmm

Reply:
Fixed! Thanks for the suggestion.